<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The Current Sgtk Environment</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="header_label">
     <property name="text">
      <string>Select any Asset in the tree and see the corresponding Tasks</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="task_list_header">
         <property name="text">
          <string>Asset Tree</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="asset_tree_view"/>
       </item>
       <item>
        <widget class="QPushButton" name="clear_cache_button">
         <property name="text">
          <string>Clear Cache</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="asset_tree_header">
         <property name="text">
          <string>Associated Tasks</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="task_list_view"/>
       </item>
       <item>
        <widget class="QCheckBox" name="dynamic_sort_checkbox">
         <property name="text">
          <string>Click to toggle setDynamicSortFilter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refresh_button">
         <property name="text">
          <string>Reload Tasks</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
